imgproc.go

```diff
@@ -1602,14 +1602,25 @@ func ApplyCustomColorMap(src Mat, dst *Mat, customColormap Mat) {
 }
 
 // GetPerspectiveTransform returns 3x3 perspective transformation for the
-// corresponding 4 point pairs.
+// corresponding 4 point pairs as image.Point.
 //
 // For further details, please see:
 // https://docs.opencv.org/master/da/d54/group__imgproc__transform.html#ga8c1ae0e3589a9d77fffc962c49b22043
 func GetPerspectiveTransform(src, dst []image.Point) Mat {
 	srcPoints := toCPoints(src)
 	dstPoints := toCPoints(dst)
-	return newMat(C.GetPerspectiveTransform(srcPoints, dstPoints))
+	return Mat{p: C.GetPerspectiveTransform(srcPoints, dstPoints)}
```

Comment:
Any reason to not use `newMat()` here to be able to use profiling?

Reply:
Good catch, I missed that